MAINTAINERS.md

```diff
@@ -42,6 +42,8 @@ If you are an owner of the organization, you can see an automated list
   [27 November 2020](https://github.com/tldr-pages/tldr/issues/4994) - present
 - **Florian Benscheidt ([@Waples](https://github.com/Waples))**:
   [16 April 2021](https://github.com/tldr-pages/tldr/issues/5774) - present
+- **Adam Herst ([@aherst](https://github.com/aherst))**:
+  [22 April 2021](https://github.com/tldr-pages/tldr/issues/5810) - present  
```

Comment:
Until defined, let's do what we've been doing and use the date that the original issue was opened.
```suggestion
  [21 April 2021](https://github.com/tldr-pages/tldr/issues/5810) - present  
```